bitnami/concourse/values.yaml

```diff
@@ -0,0 +1,1144 @@
+  ## @param web.teamSecretsPath
+  keySecretsPath: "/concourse-keys"
+  teamSecretsPath: "/team-authorized-keys"
+
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param web.podSecurityContext.enabled Enabled web pods' Security Context
+  ## @param web.podSecurityContext.fsGroup Set web pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param web.containerSecurityContext.enabled Enabled web containers' Security Context
+  ## @param web.containerSecurityContext.runAsUser Set web containers' Security Context runAsUser
+  ##
+  containerSecurityContext:
+    enabled: true
+    runAsUser: 1001
+  
+  ## @param web.podSecurityPolicy.create Specifies whether a PodSecurityPolicy should be created (set `podSecurityPolicy.create` to `true` to enable it)
+  podSecurityPolicy:
+    create: false
+
+  service: 
+    ## @param web.service.type For minAvailableikube, set this to ClusterIP, elsewhere use LoadBalancer or NodePort
+    ## Ref: https://kubernetes.io/docs/user-guide/services/#publishing-services---service-types
+    ##
+    type: ClusterIP
+
+    ## @param web.service.clusterIP When using `web.service.type: ClusterIP`, sets the user-specified cluster IP.
+    ## Example: 172.217.1.174
+    ##
+    clusterIP:
+
+    ## @param web.service.loadBalancerIP  When using `web.service.type: LoadBalancer`, sets the user-specified load balancer IP.
+    ## Example: 172.217.1.174
+    ##
+    loadBalancerIP:
+
+    ## @param web.service.labels Additional Labels to be added to the web api service.
+    ##
+    labels:
+
+    ## @param web.service.annotations Annotations to be added to the web api service.
+    ##
+    ##
+    ## When using `web.service.type: LoadBalancer` in AWS, enable HTTPS with an ACM cert:
+    ##
+    ##   service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "arn:aws:acm:eu-west-1:123456789:certificate/abc123-abc123-abc123-abc123"
+    ##   service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
+    ##   service.beta.kubernetes.io/aws-load-balancer-backend-port: "api"
+    ##   service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+    ##
+    annotations: {}
+
+    ## @param web.service.loadBalancerSourceRanges When using `web.service.type: LoadBalancer`, restrict access to the load balancer to particular IPs
+    ## Example:
+    ##   - 192.168.1.10/32
+    ##
+    loadBalancerSourceRanges:
+
+    ## @param web.service.NodePort When using `web.service.type: NodePort`, sets the nodePort for api
+    ##
+    NodePort:
+
+    ## @param web.service.tlsNodePort When using `web.service.type: NodePort`, sets the nodePort for api tls
+    ##
+    tlsNodePort:
+    
+    workerGateway:
```

Comment:
In order to maintain some consistency, I'd rather have something like `web.service.web` and `web.service.workerGateway`. Or, as we only have two services to expose, maybe you can change it to `service.web` and `service.workerGateway`